### DISCUSSION TITLE
[BP#46] Limiter le nombre de requêtes HTTP

### DISCUSSION THREAD

**Discussion by ldevernay** [cnumr/best-practices#547] [Bonnes pratiques] [answered]
Il pourrait aussi être intéressant de mentionner les requêtes HTTP générées par l'usage : autocomplétion et autres.
Aussi, je ne suis pas sûr que les sprites d'images restent avantageux pour l'écoconception je regarderai la BP dédiée), de même pour la police d'icônes (qui, si elle est utilisée, ne devrait inclure que les icônes utilisées).

**Comment by rlemaire** [comment, marked as answer]
Il y a d'autres fiches pour chacun de ces points.
